<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>896</width>
    <height>749</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <weight>50</weight>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>比特币账户查询</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>901</width>
      <height>641</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="tab1">
     <attribute name="title">
      <string>生成Testnet账户</string>
     </attribute>
     <widget class="QPushButton" name="ButtonForUser">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>260</y>
        <width>151</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Agency FB</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>生成账户</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_create_user">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>260</y>
        <width>441</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>16</pointsize>
       </font>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab2">
     <attribute name="title">
      <string>查询Testnet账户</string>
     </attribute>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>150</y>
        <width>491</width>
        <height>101</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="font">
          <font>
           <family>宋体</family>
           <pointsize>26</pointsize>
          </font>
         </property>
         <property name="text">
          <string>查询结果</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QFormLayout" name="formLayout_3">
         <item row="0" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="font">
              <font>
               <family>黑体</family>
               <pointsize>14</pointsize>
               <weight>75</weight>
               <bold>true</bold>
               <strikeout>false</strikeout>
              </font>
             </property>
             <property name="text">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#00aa00;&quot;&gt;0&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_3">
             <property name="font">
              <font>
               <family>宋体</family>
               <pointsize>14</pointsize>
              </font>
             </property>
             <property name="text">
              <string>satoshi</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_4">
           <property name="font">
            <font>
             <family>宋体</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>账户余额：</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="formLayoutWidget">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>30</y>
        <width>491</width>
        <height>107</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>地址</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="lineEdit">
         <property name="font">
          <font>
           <family>Times New Roman</family>
           <pointsize>15</pointsize>
          </font>
         </property>
         <property name="text">
          <string>n1N4gibGTZyfovfoK6e85g9XoncAgrFATt</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="pushButton">
         <property name="font">
          <font>
           <pointsize>15</pointsize>
          </font>
         </property>
         <property name="text">
          <string>查询</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QTreeWidget" name="treeWidget">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>250</y>
        <width>491</width>
        <height>211</height>
       </rect>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectItems</enum>
      </property>
      <property name="headerHidden">
       <bool>false</bool>
      </property>
      <column>
       <property name="text">
        <string>UTXO</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Value</string>
       </property>
      </column>
     </widget>
    </widget>
    <widget class="QWidget" name="tab3">
     <attribute name="title">
      <string>Testnet转账</string>
     </attribute>
     <widget class="QPushButton" name="transport">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>300</y>
        <width>221</width>
        <height>34</height>
       </rect>
      </property>
      <property name="text">
       <string>转账</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>160</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>新宋体</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>转账地址</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_transport">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>150</y>
        <width>421</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="trans_money">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>210</y>
        <width>421</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>200</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>新宋体</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>转账金额</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab4">
     <attribute name="title">
      <string>生成比特币账户</string>
     </attribute>
     <widget class="QLineEdit" name="lineEdit_create_user_real">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>260</y>
        <width>411</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>16</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QPushButton" name="ButtonForUserReal">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>260</y>
        <width>141</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Agency FB</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>生成账户</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab5">
     <attribute name="title">
      <string>查询比特币账户</string>
     </attribute>
     <widget class="QWidget" name="verticalLayoutWidget_4">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>170</y>
        <width>491</width>
        <height>101</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="label_22">
         <property name="font">
          <font>
           <family>宋体</family>
           <pointsize>26</pointsize>
          </font>
         </property>
         <property name="text">
          <string>查询结果</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QFormLayout" name="formLayout_7">
         <item row="0" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QLabel" name="label_23">
             <property name="font">
              <font>
               <family>黑体</family>
               <pointsize>14</pointsize>
               <weight>75</weight>
               <bold>true</bold>
               <strikeout>false</strikeout>
              </font>
             </property>
             <property name="text">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#00aa00;&quot;&gt;0&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_24">
             <property name="font">
              <font>
               <family>宋体</family>
               <pointsize>14</pointsize>
              </font>
             </property>
             <property name="text">
              <string>satoshi</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_25">
           <property name="font">
            <font>
             <family>宋体</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>账户余额：</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="formLayoutWidget_4">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>50</y>
        <width>491</width>
        <height>107</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="formLayout_8">
       <item row="0" column="0">
        <widget class="QLabel" name="label_26">
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>地址</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="lineEdit_4">
         <property name="font">
          <font>
           <family>Times New Roman</family>
           <pointsize>15</pointsize>
          </font>
         </property>
         <property name="text">
          <string>18RKfuiBLFAq86dwyfUmX5gvFFZFYSMwgw</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="pushButton_real">
         <property name="font">
          <font>
           <pointsize>15</pointsize>
          </font>
         </property>
         <property name="text">
          <string>查询</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QTreeWidget" name="treeWidget_4">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>270</y>
        <width>491</width>
        <height>211</height>
       </rect>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectItems</enum>
      </property>
      <property name="headerHidden">
       <bool>false</bool>
      </property>
      <column>
       <property name="text">
        <string>UTXO</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Value</string>
       </property>
      </column>
     </widget>
    </widget>
    <widget class="QWidget" name="tab6">
     <attribute name="title">
      <string>转账比特币账户</string>
     </attribute>
     <widget class="QPushButton" name="transport_4">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>310</y>
        <width>221</width>
        <height>34</height>
       </rect>
      </property>
      <property name="text">
       <string>转账</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_27">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>80</y>
        <width>161</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>新宋体</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>转账地址</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_transport_4">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>100</y>
        <width>421</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="trans_money_4">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>190</y>
        <width>421</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_28">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>180</y>
        <width>131</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>新宋体</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>转账金额</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
